src/generate_fastapi/mako_templates/src/project/domain/model/template_schemas.py

```diff
@@ -27,11 +27,9 @@ class _Base(pydantic.BaseModel):
 % for field in model.fields:
     % if not field.is_primary_key:
     % if field.field_type.nullable:
-    ${field.name}: Optional[${JsonTranslator.translate_typename_to_pytypes(field.field_type.name)}] = pydantic.Field(None${', max_length='+str(field.field_type.max_length) if field.field_type.max_length else ''}
-                                                                                                                     ${', min_length='+str(field.field_type.min_length) if field.field_type.min_length else ''})
+    ${field.name}: Optional[${JsonTranslator.translate_typename_to_pytypes(field.field_type.name)}] = pydantic.Field(None${', max_length='+str(field.field_type.max_length) if field.field_type.max_length else ''}${', min_length='+str(field.field_type.min_length) if field.field_type.min_length else ''})
```

Comment:
What's the reasoning behind making the lines so long? Is it due to linting after someone uses the generator?

Reply:
I could change it to something like:
`          ${field.name}: /
Optional[${JsonTranslator.translate_typename_to_pytypes(field.field_type.name)}] = /
pydantic.Field(None${', max_length='+str(field.field_type.max_length) if field.field_type.max_length else ''}/
${', min_length='+str(field.field_type.min_length) if field.field_type.min_length else ''})`
As "/" will evaluate all the lines into one line. However due to the mako read this, there cannot be indentation on the next line as this will translate with it as well.

For example:
`SomeCode():
       here is a line that goes onto \
       another line.
`
would be evaluated to:
`SomeCode():
       here is a line that goes onto        another line.
`
I am not certain what is most readable?